src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,12 +1,15 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
The `@Component` annotation is typically used for classes that are intended to be managed by a dependency injection container. Applying it to an interface like `ProductParser` is not appropriate. Consider removing the `@Component` annotation from the interface.

Reply:
changes have been made